bigframes/bigquery/__init__.py

```diff
@@ -148,3 +152,153 @@ def array_to_string(series: series.Series, delimiter: str) -> series.Series:
 
     """
     return series._apply_unary_op(ops.ArrayToStringOp(delimiter=delimiter))
+
+
+def vector_search(
+    base_table: str,
+    column_to_search: str,
+    query: Union[dataframe.DataFrame, series.Series],
+    *,
+    query_column_to_search: Optional[str] = None,
+    top_k: Optional[int] = 10,
+    distance_type: Literal["euclidean", "cosine"] = "euclidean",
+    fraction_lists_to_search: Optional[float] = None,
+    use_brute_force: bool = False,
+) -> dataframe.DataFrame:
+    """
+    Conduct vector search which searches embeddings to find semantically similar entities.
+
+    **Examples:**
+
+
+        >>> import bigframes.pandas as bpd
+        >>> import bigframes.bigquery as bbq
+        >>> bpd.options.display.progress_bar = None
+
+    DataFrame embeddings for which to find nearest neighbors, and ARRAY<FLOAT> column
```

Comment:
Let's use code font for `ARRAY<FLOAT>`. Also the BQ type is called `FLOAT64` in standard SQL https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#floating_point_types

```suggestion
    DataFrame embeddings for which to find nearest neighbors. The ``ARRAY<FLOAT64>`` column
```